### PR TITLE
frontend: Add breadcrumbs component

### PR DESCRIPTION
**Pull request by septum**
### Description

- Added Breadcrumbs component

#### PRs

- Relies on #3143 

#### Before

localhost_3000_catalog_clutch

#### After

localhost_3000_catalog

### Testing Performed

Manual

### CODE REVIEW

**Review by jdslaugh — COMMENTED** [review]
Looks good! From the image it looks like the last entry is getting the link applied, is that happening? I assume it's from test data but want to make sure.

Also, is this actually reliant on `workflow-layout`? Don't see anything related to it. So it could go in separately.

**Comment by septum** [comment]
> Looks good! From the image it looks like the last entry is getting the link applied, is that happening? I assume it's from test data but want to make sure.
> 
> Also, is this actually reliant on `workflow-layout`? Don't see anything related to it. So it could go in separately.

It was used for testing only, I'll make sure to retest after the merge of #3143

File: frontend/packages/core/src/Breadcrumbs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Breadcrumbs as MuiBreadcrumbs, Theme } from "@mui/material";
+import { alpha } from "@mui/system";
+
+import styled from "../styled";
+
+import Breadcrumb from "./breadcrumb";
+import { BreadcrumbEntry } from "./types";
```
Comment (jecr) [inline frontend/packages/core/src/Breadcrumbs/index.tsx:8]:
```suggestion
import type { BreadcrumbEntry } from "./types";
```

File: frontend/packages/core/src/Breadcrumbs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const Breadcrumbs = ({ entries }: BreadcrumbsProps) => {
+  return (
+    <StyledBreadcrumbs>
+      {entries.map((entry: BreadcrumbEntry) => (
+        <Breadcrumb key={entry.label} {...entry} />
+      ))}
+    </StyledBreadcrumbs>
+  );
+};
```
Comment (jecr) [inline frontend/packages/core/src/Breadcrumbs/index.tsx:29]: nit
```suggestion
const Breadcrumbs = ({ entries }: BreadcrumbsProps) => (
    <StyledBreadcrumbs>
      {entries.map((entry: BreadcrumbEntry) => (
        <Breadcrumb key={entry.label} {...entry} />
      ))}
    </StyledBreadcrumbs>
  );
```

**Review by jecr — APPROVED** [review]
Left a few minor comments

**Review by jdslaugh — APPROVED** [review]

**Pull request merged**